_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -3357,35 +3357,35 @@ sections above.
 1.1,<<security_questions>>,Complete the contact details on the AWS account page
 1.2,<<security_questions>>,Complete the security contact information on the AWS account page
 1.3,<<security_questions>>,Answer the security questions on the AWS account page
-1.4,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that no access key exists for the root user
+1.4,<<configure_security_hub>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that no access key exists for the root user
 1.5,<<root_mfa>>,Manually configure MFA for the root user
 1.6,<<root_mfa>>,Use a Yubikey (or other hardware MFA) for the root user
 1.7,<<manual_steps>>,Take manual steps to complete this recommendation
-1.8-9,<<create_password_policy>>,Use the IAM password policy module
+1.8-9,<<apply_account_baseline_security>>,Use the `account-baseline-security` module to create users
 1.10,<<configure_authentication>>,Configure authentication using SAML or IAM
-1.11,<<iam_user_authentication>>,Create IAM users with the `iam-users` module
-1.12,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused credentials
-1.13,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no extra access keys
-1.14,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused access keys
-1.15,<<iam_user_authentication>>,Use IAM groups
-1.16,<<iam_user_authentication>>,Use the Gruntwork modules to create best-practices groups and roles
-1.17,<<iam_user_authentication>>,Use the `iam-groups` module to create a support group
+1.11,<<apply_account_baseline_security>>,Use the `account-baseline-security` module to create users
+1.12,<<configure_security_hub>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused credentials
```

Comment:
```suggestion
1.12,<<apply_account_baseline_security>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused credentials
```

Reply:
I believe the Security Hub monitors this, but we have an AWS Config rule for this that should get kicked in

Reply:
This is a bit tricky, since those are links pointing to the specific sections in the guide, so `apply_account_baseline_all` won't work. I guess I could put multiple links pointing to each account baseline module